.github/workflows/terraform.yaml

```diff
@@ -138,7 +138,7 @@ jobs:
           key: ${{ runner.os }}-tflint-${{ hashFiles('.tflint.hcl') }}
 
       - name: Run tflint with review comment on PR
-        uses: reviewdog/action-tflint@master
+        uses: reviewdog/action-tflint@v1.17.3
```

Comment:
@thepoppingone I think we need to add `tflint_version` instead.. which will help to download a specific version instead of https://api.github.com/repos/terraform-linters/tflint/releases/latest

Reply:
I think this PR is not relevant now since the tflint ignore one already has it